src/db/migrations/20190728221239-create-notification.js

```diff
@@ -0,0 +1,51 @@
+import Debug from 'debug';
+
+const debug = Debug('dev');
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    try {
+      await queryInterface.createTable('Notification', {
+        id: {
+          allowNull: false,
+          primaryKey: true,
+          type: Sequelize.UUID
+        },
+        userId: {
+          allowNull: false,
+          primaryKey: true,
+          type: Sequelize.UUID
+        },
+        event: {
+          allowNull: false,
+          type: Sequelize.STRING
+        },
+        payload: {
+          allowNull: false,
+          type: Sequelize.JSON
+        },
+        notified: {
+          allowNull: false,
+          type: Sequelize.BOOLEAN
+        },
+        createdAt: {
+          allowNull: false,
+          type: Sequelize.DATE
+        },
+        updatedAt: {
+          allowNull: false,
+          type: Sequelize.DATE
+        }
+      });
+    } catch (error) {
+      debug(error);
+    }
+  },
+  down: async (queryInterface) => {
+    try {
+      await queryInterface.dropTable('AuNotificationthor');
```

Comment:
??? I think you should actually run and undo the migrations to catch errors like this one
```suggestion
      await queryInterface.dropTable('Notifications');
```

Reply:
done